catalog/transformer/catalog-transformer-preview/src/main/java/org/codice/ddf/transformer/preview/actions/PreviewActionProvider.java

```diff
@@ -60,10 +60,7 @@ public PreviewActionProvider(String actionProviderId) {
   @Override
   protected boolean canHandleMetacard(Metacard metacard) {
     Attribute bodyText = metacard.getAttribute(Extracted.EXTRACTED_TEXT);
-    if (bodyText != null && StringUtils.isNotBlank((String) bodyText.getValue())) {
-      return true;
-    }
-    return false;
+    return bodyText != null && StringUtils.isNotBlank((String) bodyText.getValue());
```

Comment:
the old way was more clear

Reply:
I disagree. This reads like "If the bodyText is not null and the bodyText value is not blank". Then you look at the method name and it's `canHandleMetacard`.

Reply:
you should at least add a javadoc comment explaining that this method can return true OR false

Reply:
A boolean can be true or false.  I'm not sure i understand the concern

Reply:
what if this method returns both though?